integration/containerd/confidential/agent_image.bats

```diff
@@ -105,10 +106,20 @@ setup() {
 	create_test_pod
 
 	assert_container_fail "$container_config"
-	assert_logs_contain 'Validate image failed: \[PublicKeyVerifier { key: ECDSA_P256_SHA256_ASN1'
+	assert_logs_contain "kata" 'Validate image failed: \[PublicKeyVerifier { key: ECDSA_P256_SHA256_ASN1'
 }
 
 @test "$test_tag Test pull an unencrypted unsigned image from an authenticated registry with correct credentials" {
+	cat << EOF | tee ~/.docker/config
+{
+  "auths": {
+    "quay.io": {
+      "auth": "a2F0YS1jb250YWluZXJzK2NjX2F1dGg6VjFBRkU5UkM0TVQyQTZNR0pCVTIxUFJON1VETkFHMVNMTDJNOVNMUVlLOEE5VExLR09WN04wUlNYNUw0RTgxVA==",
```

Comment:
Can we use `$REGISTRY_CREDENTIAL_ENCODED` please rather than leaking the credentials? (which we might need to re-gen now)